README.md

```diff
@@ -1,21 +1,39 @@
-<!--cspell:ignore sarg  -->
+<!--cspell:ignore sarg trivy gitui kubectx Fira Firacode Caskaydia Consolas   -->
 # Go Dev Container
 
-<!-- [![OpenSSF Scorecard](https://api.scorecard.dev/projects/github.com/sarg3nt/cert-manager-webhook-infoblox-wapi/badge)](https://scorecard.dev/viewer/?uri=github.com/sarg3nt/cert-manager-webhook-infoblox-wapi)
-[![CodeQL](https://github.com/sarg3nt/cert-manager-webhook-infoblox-wapi/actions/workflows/github-code-scanning/codeql/badge.svg)](https://github.com/sarg3nt/cert-manager-webhook-infoblox-wapi/actions/workflows/github-code-scanning/codeql)
-[![Release](https://github.com/sarg3nt/cert-manager-webhook-infoblox-wapi/actions/workflows/release.yml/badge.svg)](https://github.com/sarg3nt/cert-manager-webhook-infoblox-wapi/actions/workflows/release.yml) -->
-
-A Go Dev Container using `mise` to install Go and other conventinet tools.  `mise` can then be used to install various other go versions as needed.
-
+[![OpenSSF Scorecard](https://api.scorecard.dev/projects/github.com/sarg3nt/go-dev-container/badge)](https://scorecard.dev/viewer/?uri=github.com/sarg3nt/go-dev-container)
+[![Scorecard supply-chain security](https://github.com/sarg3nt/go-dev-container/actions/workflows/scorecard.yml/badge.svg)](https://github.com/sarg3nt/go-dev-container/actions/workflows/scorecard.yml)
+[![trivy](https://github.com/sarg3nt/go-dev-container/actions/workflows/trivy.yml/badge.svg)](https://github.com/sarg3nt/go-dev-container/actions/workflows/trivy.yml)
+[![Dependabot Updates](https://github.com/sarg3nt/go-dev-container/actions/workflows/dependabot/dependabot-updates/badge.svg)](https://github.com/sarg3nt/go-dev-container/actions/workflows/dependabot/dependabot-updates)
+[![Dependency Review](https://github.com/sarg3nt/go-dev-container/actions/workflows/dependency-review.yml/badge.svg)](https://github.com/sarg3nt/go-dev-container/actions/workflows/dependency-review.yml)
+[![Release](https://github.com/sarg3nt/go-dev-container/actions/workflows/release.yml/badge.svg)](https://github.com/sarg3nt/go-dev-container/actions/workflows/release.yml)
+
+A Go Dev Container using `mise` to install Go and other convenient tools.  
+`mise` can then be used to install various other Go versions as needed.
+
+- [Included Tools](#included-tools)
+  - [Go](#go)
+  - [Utilities](#utilities)
+- [Using `mise` to Manage Go Versions](#using-mise-to-manage-go-versions)
+- [Included `.devcontainer` Config](#included-devcontainer-config)
+  - [Dev Container Setup](#dev-container-setup)
+  - [`dev.sh`](#devsh)
+- [Starship](#starship)
+- [Initial Worksation Setup](#initial-worksation-setup)
```

Comment:
Typo for 'workstation'? There seems to be 8 instances in total in this file.

Reply:
Fixed all